site/content/docs/5.2/components/pagination.md

```diff
@@ -121,12 +121,68 @@ Or with `.justify-content-end`:
 <!-- Boosted mod -->
 ## Responsive behavior
 
-To avoid pagination wrapping or overflowing, we limit the number or pagination item to display depending on breakpoint.
+By default, every item is visible. Here is an example with our basic responsive behavior.
 
 {{< example >}}
 <nav aria-label="Page navigation example">
   <ul class="pagination">
-    <li class="page-item disabled"><a class="page-link">Previous</a></li>
+    <li class="page-item"><a class="page-link" href="#" aria-label="Previous"></a></li>
+    <li class="page-item"><a class="page-link" href="#">1</a></li>
+    <li class="page-item"><a class="page-link" href="#">2</a></li>
+    <li class="page-item"><a class="page-link" href="#">3</a></li>
+    <li class="page-item"><a class="page-link" href="#">4</a></li>
+    <li class="page-item active" aria-current="page">
+      <a class="page-link" href="#">5</a>
+    </li>
+    <li class="page-item"><a class="page-link" href="#">6</a></li>
+    <li class="page-item"><a class="page-link" href="#">7</a></li>
+    <li class="page-item"><a class="page-link" href="#">8</a></li>
+    <li class="page-item"><a class="page-link" href="#">9</a></li>
+    <li class="page-item"><a class="page-link" href="#">10</a></li>
+    <li class="page-item"><a class="page-link" href="#" aria-label="Next"></a></li>
+  </ul>
+</nav>
+{{< /example >}}
+
+{{< callout warning >}}
+**Heads up—our smart responsive pagination is opt-in as of v5.2.1!**
+
+It means that it is disabled by default in order to have a lighter CSS bundle. We included it here in our documentation's CSS for this specific use case just as a demonstration of the rendering.
+
+<details class="border-top border-bottom border-1 border-light">
+<summary class="fw-bold p-2 mb-0">
+{{< markdown >}}
+Enable the responsive pagination by setting `$enable-responsive-pagination: true;`
+{{< /markdown >}}
+</summary>
+{{< markdown >}}
+
+```scss
+// Required
+@import "../node_modules/boosted/scss/functions";
+
+// Activate responsive pagination
+$enable-responsive-pagination: true;
+
+// Required
+@import "../node_modules/boosted/scss/variables";
+@import "../node_modules/boosted/scss/mixins";
+@import "../node_modules/boosted/scss/grid";
+
+// Optional Boosted components here
+@import "../node_modules/boosted/scss/pagination";
+```
+Then, recompile your Sass.
+{{< /markdown >}}
+</details>
+{{< /callout >}}
+
+The following example uses our smart responsive pagination. To avoid pagination wrapping or overflowing, we limit the number of displayed pagination items depending on the breakpoints by displaying automatically some ellipsis.
+
+{{< example class="bd-example-responsive-pagination px-0" >}}
```

Comment:
Because of `.px-0`:
![screenshot-deploy-preview-1370--boosted netlify app-2022 12 20-10_13_51](https://user-images.githubusercontent.com/92363071/208629772-69ca2c7f-c837-41ac-a6ce-87cf7cfbc516.png)